modules/i3s/src/i3s-loader.ts

```diff
@@ -38,6 +39,13 @@ export const I3SLoader = {
 async function parseI3S(data, options, context, loader) {
   const url = context.url;
   options.i3s = options.i3s || {};
+  const magicNumber = getMagicNumber(data);
+
+  // check if file is slpk
+  if (magicNumber === SLPK_HEX) {
+    console.warn('Files with .slpk extention currently are not supported by I3SLoader'); // eslint-disable-line
```

Comment:
Should we `throw new Error()` instead?

Check how we handle other errors  in loaders - it should be consistent.

Reply:
> Should we `throw new Error()` instead?

Ok, fixed :)